packages/data-transforms/src/track-value-array-to-segments/track-value-array-to-segments.ts

```diff
@@ -3,12 +3,36 @@ import { Field, Segment, Point } from './types'
 export const TRACK_FIELDS = [Field.lonlat, Field.timestamp, Field.speed]
 export const DEFAULT_NULL_VALUE = -Math.pow(2, 31)
 
+/**
+ * Transformer functions to be applied on every valueArray field
+ * to convert the received value (Int32) to the actual value (Float)
+ * depending on its precision.
+ * This transformations should be the inverse of the formatterValueArray
+ * property of every field defined in:
+ *  https://github.com/GlobalFishingWatch/api-vessels-service/blob/develop/src/modules/api/v1/vessels/modules/tracks/services/feature.service.ts
+ */
+const transformerByField: Partial<Record<Field, (value: number) => number>> = {
+  [Field.course]: (value: number) => value / Math.pow(10, 6),
+  [Field.distanceFromPort]: (value: number) => value,
+  // Uncomment when added to new package version
+  // [Field.distanceFromShore]: (value: number) => value,
+  [Field.elevation]: (value: number) => value,
+  // Uncomment when added to new package version
+  // [Field.encounter]: (value: number) => value,
+  [Field.latitude]: (value: number) => value / Math.pow(10, 6),
+  [Field.longitude]: (value: number) => value / Math.pow(10, 6),
+  [Field.fishing]: (value: number) => value,
```

Comment:
I think this is wrong, fishing is a value from 0 to 1 with decimals

Reply:
No, it is correct. Fishing is an integer, 0 or 1 without decimals. At least that's how it's encoded (https://github.com/GlobalFishingWatch/api-vessels-service/blob/develop/src/modules/api/v1/vessels/modules/tracks/services/feature.service.ts#L17-L24)